chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/test/InstanceSupervisor.kt

```diff
@@ -256,37 +253,37 @@ object InstanceActions {
                         body = mapOf(
                                 "name" to name,
                                 "id" to id,
-                                "avatar_url" to avatarUrl,
-                                "custom_data" to customData
+                                "avatar_url" to avatarUrl
+//                                "custom_data" to customData
                         ).toJson()
                 ),
                 tokenProvider = sudoTokenProvider,
                 responseParser = { it.parseAs() }
         )
     }.withName("Create new user: $id")
 
-    fun changeRoomName(room: Room, newName: String): InstanceAction = {
-        chatkitInstance.request<JsonElement>(
-                options = RequestOptions(
-                        path = "/rooms/${room.id}",
-                        method = "PUT",
-                        body = mapOf("name" to newName).toJson()
-                ),
-                tokenProvider = sudoTokenProvider,
-                responseParser = { it.parseAs() }
-        )
-    }.withName("Changing name of room ${room.name} to $newName ")
-
-    fun deleteRoom(room: Room): InstanceAction = {
-        chatkitInstanceV2.request<JsonElement>(
-                options = RequestOptions(
-                        path = "/rooms/${room.id}",
-                        method = "DELETE"
-                ),
-                tokenProvider = sudoTokenProvider,
-                responseParser = { it.parseAs() }
-        )
-    }.withName("Deleting room ${room.name} ")
+//    fun changeRoomName(room: Room, newName: String): InstanceAction = {
+//        chatkitInstance.request<JsonElement>(
+//                options = RequestOptions(
+//                        path = "/rooms/${room.id}",
+//                        method = "PUT",
+//                        body = mapOf("name" to newName).toJson()
+//                ),
+//                tokenProvider = sudoTokenProvider,
+//                responseParser = { it.parseAs() }
+//        )
+//    }.withName("Changing name of room ${room.name} to $newName ")
+
+//    fun deleteRoom(room: Room): InstanceAction = {
+//        chatkitInstanceV2.request<JsonElement>(
+//                options = RequestOptions(
+//                        path = "/rooms/${room.id}",
+//                        method = "DELETE"
+//                ),
+//                tokenProvider = sudoTokenProvider,
+//                responseParser = { it.parseAs() }
+//        )
+//    }.withName("Deleting room ${room.name} ")
```

Comment:
for deleteRoom it would be better to keep it just change it to accept roomId (and also use it for in withName as that's always there, as opposed to room.name which may be null)

Reply:
That makes sense - have updated the comment to read `.withName("Deleting room ${roomId} ")`. The [function preceding this](chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/test/InstanceSupervisor.kt) is `changeRoomName` - i could update that to also only take in a String for the roomId? What do you think?

Reply:
Cool, thanks :) Regarding `changeRoomName` after a deeper look I think it's fine to remove it altogether. After all we will aim to cover more at lower level tests and have only core scenarios e2e tested. The test that used it was indeed testing more the backend than the SDK. Hence likewise I think it's fine to remove `deleteRoom` here.

Reply:
Yep! that makes sense. Have removed `changeRoomName`!